docs/building-deployer-helm.md

```diff
@@ -107,13 +113,20 @@ metadata:
 spec:
   descriptor:
     type: Wordpress
-    version: '6'
+    version: '6.0.0'
   selector:
     matchLabels:
       app.kubernetes.io/name: "{{ .Release.Name }}"
+  addOwnerRef: true
   componentKinds:
-  - group: batch/v1
-    kind: Job
+  - group: v1
+    kind: PersistentVolumeClaim
+  - group: v1
+    kind: Secret
+  - group: v1
+    kind: Service
+  - group: apps/v1
+    kind: Deployment
```

Comment:
Drop versions form these groups. So for core objects, the groups are empty. Specifically: 

```suggestion
  - group: ''
    kind: PersistentVolumeClaim
  - group: ''
    kind: Secret
  - group: ''
    kind: Service
  - group: apps
    kind: Deployment
```